src/plugins/intel_gpu/src/graph/select.cpp

```diff
@@ -55,6 +55,28 @@ std::vector<layout> select_inst::calc_output_layouts(const select_node& /*node*/
     return {{output_shapes[0], dt, format::get_default_format(output_shapes[0].size())}};
 }
 
+std::vector<size_t> select_inst::extend_input_shape_to_6d(kernel_impl_params const& orig_impl_param, int32_t input_idx) {
+    ov::PartialShape ps = orig_impl_param.get_input_layout(input_idx).get_partial_shape();
+
+    if (ps.size() < 4) {
+        ps.insert(ps.begin(), 4 - ps.size(), ov::Dimension(1));
+    }
+
+    layout l(ps, data_types::i32, format::get_default_format(ps.size()));
+    return l.transform(format::bfwzyx).to_shape();
+}
+
+std::vector<size_t> select_inst::extend_output_shape_to_6d(kernel_impl_params const& orig_impl_param, int32_t output_idx) {
```

Comment:
`extend_output_shape_to_6d()` and `extend_input_shape_to_6d()` functions look mostly identical, so maybe it makes sense to extract common part to `extend_shape_to_6d()` function and reuse the code